apps/next-evans/pages/[friend]/[friend_slug].tsx

```diff
@@ -0,0 +1,215 @@
+import { PrismaClient } from '@prisma/client';
+import invariant from 'tiny-invariant';
+import cx from 'classnames';
+import { t } from '@friends-library/locale';
+import type { GetStaticPaths, GetStaticProps } from 'next';
+import type { Edition } from '@/lib/editions';
+import { LANG } from '@/lib/env';
+import { mostModernEdition } from '@/lib/editions';
+import FriendBlock from '@/components/pages/friend/FriendBlock';
+import FeaturedQuoteBlock from '@/components/pages/friend/FeaturedQuoteBlock';
+import BookByFriend from '@/components/pages/friend/BookByFriend';
+import TestimonialsBlock from '@/components/pages/friend/TestimonialsBlock';
+
+const client = new PrismaClient();
+
+export const getStaticPaths: GetStaticPaths = async () => {
+  const allFriends = await client.friends.findMany({
+    where: { lang: LANG },
+    select: { slug: true, gender: true },
+  });
+
+  const paths = allFriends.map((friend) => {
+    if (LANG === `en`) {
+      return {
+        params: { friend: `friend`, friend_slug: friend.slug },
+      };
+    }
+    return {
+      params: {
+        friend: friend.gender === `female` ? `amiga` : `amigo`,
+        friend_slug: friend.slug,
```

Comment:
you could (and probably should) punt this to another PR, but i just remembered that the special compilations "friends" has a unique url... `/compilations` or `/compilaciones` -- not two parts like all the rest. not sure how we'll pull that off. might need to do a server redirect or something fancy.  but maybe put it on your list to investigate...